src/nps/userSurvey.ts

```diff
@@ -228,7 +225,33 @@ class UserSurveyWebviewController extends ReactWebviewPanelController<
 
         this.registerReducer("submit", async (state, payload) => {
             this._onSubmit.fire(payload.answers);
+
             this.panel.dispose();
+
+            if (
+                (payload.answers.nps as number) < 7 /* NPS detractor */ ||
+                (payload.answers.nsat as number) < 2 /* NSAT dissatisfied */
+            ) {
+                const response = await vscode.window.showInformationMessage(
+                    locConstants.UserSurvey.fileABugPrompt,
+                    locConstants.UserSurvey.submitBug,
+                    locConstants.Common.cancel,
+                );
+
+                if (response === locConstants.UserSurvey.submitBug) {
+                    const encodedIssueBody = encodeURIComponent(
```

Comment:
should we log a telemetry event here to understand if anyone is using this option?

Reply:
added